src/lib/remark-plugins/remark-plugin-adjust-link-urls/helpers.ts

```diff
@@ -94,10 +151,10 @@ const preAdjustUrl = ({ currentPath, url }): string => {
 		return url
 	}
 
-	// Do nothing if url is not an internal path
-	const isInternalPath = getIsInternalPath(url)
-	if (!isInternalPath) {
-		return url
+	// Handle full URL that may link externally or internally
+	const isFullUrl = getIsInternalPath(url) === false
+	if (isFullUrl) {
+		return handleFullUrl({ url })
```

Comment:
Not sure if you need to pass it as an object?

Reply:
Just doing it for consistency with the majority of the other helpers in this file. 😊

I've found that passing a single object param rather than many different types of params has are easier to work with long term. When adding new options, they don't change the API in such a way that other calls to the function have to be refactored.